cli/integrity/integrity.go

```diff
@@ -0,0 +1,31 @@
+package integrity
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// Signer implements high-level API for operations that require
+// a private key.
+type Signer interface {
+	// AddIntegrityHashes generates files with hashes for integrity
+	// checks and signs them.
+	Sign(appList []util.AppListEntry, baseDir string) error
+}
+
+// DummySigner is a noop integrity check preparator.
+type DummySigner struct{}
+
+// NewSigner contructrs a noop Signer.
+func NewSigner(string) (Signer, error) {
+	return &DummySigner{}, nil
+}
+
+// RegisterWithIntegrityFlag is a noop function that is intended to add
+// flags to `tt pack` command.
+func RegisterWithIntegrityFlag(cmd *cobra.Command, dst *string) {}
+
+// Sign is a noop preparation of integrity check files.
+func (*DummySigner) Sign([]util.AppListEntry, string) error {
+	return nil
+}
```

Comment:
I don't see a reason to make it public.
```suggestion
// dummySigner is a noop integrity check preparator.
type dummySigner struct{}

// Sign is a noop preparation of integrity check files.
func (dummySigner) Sign([]util.AppListEntry, string) error {
	return nil
}

// NewSigner contructrs a noop Signer.
func NewSigner(string) (Signer, error) {
	return &dummySigner{}, nil
}

// RegisterWithIntegrityFlag is a noop function that is intended to add
// flags to `tt pack` command.
func RegisterWithIntegrityFlag(cmd *cobra.Command, dst *string) {}
```

Reply:
Done